components/table/src/overrides/caption.js

```diff
@@ -1,13 +1,14 @@
 /** @jsx jsx */
 
-import { jsx, useBrand } from '@westpac/core';
+import { jsx, useBrand, getLabel } from '@westpac/core';
 
 const Caption = ({ state, ...rest }) => <caption {...rest} />;
 
 const captionStyles = () => {
 	const { TYPE } = useBrand();
 
 	return {
+		label: getLabel('caption'),
```

Comment:
This should be table-caption
```suggestion
		label: getLabel('table-caption'),
```